### DISCUSSION TITLE
امکان تعیین چند کاربره کردن اکانت

### DISCUSSION THREAD

**Discussion by alijasimy** [hiddify/Hiddify-Manager#4685] [Ideas ایدهها]
**سلام و عرض ادب و خسته نباشید خدمت تیم عالی هدیفای**
در درجه اول تشکر میکنم بابت بروزرسانی های مداوم اسکریپت و خسته نباشید میگم به تیم عالی تون
ممنون میشم امکان تعیین چند کاربره کردن اکانت رو قرار بدید که بشه کاری کرد که کاربر  یا نتونه به صورت همزمان وصل بشه و همچنین تعیین کرد دو الی چند نفر به صورت همزمان وصل بشن

متشکرم

**Comment by roseshayan** [comment]
درود دوست عزیز
این کار باعث فشار بسیار سنگینی روی CPU سرورتون میشه، به همین جهت اکثرا روی همچین پلتفرم هایی این قابلیت رو نمیزارن مگر به نیاز کاربر مثل صنائی یا مرزبان به صورت اسکریپتی خود کاربران بنویسن.

**Reply by alijasimy** [reply]
بله صد در صد هر امکاناتی اضافه بشه مقداری هم از پردازش سرور رو درگیر خواهد کرد ولی ای کاش این طراحی بشه و با کلید فعال و غیرفعال باشه 
چون من خیلی از همکاران رو دیدم به خاطر این موضوع از هدیفای به سمت اسکریپت های دیگه رفتن فقط و فقط به خاطر اینکه این ویزگی در هدیفای نیست
